testing/trino-product-tests/src/main/java/io/trino/tests/iceberg/TestSparkCompatibility.java

```diff
@@ -406,14 +407,31 @@ public void testIdBasedFieldMapping()
         assertEquals(result.column(1).get(0), expected.getValues().get(0));
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testTrinoShowingSparkCreatedTables()
+    {
+        String pattern = "test_table_listing_for_%";
+        String sparkTable = pattern.replace("%", "spark");
+        String trinoTable = pattern.replace("%", "trino");
```

Comment:
you can simplify

pattern -> prefix
`.replace("%", "` -> concatenation

you can also inline all these names without any harm